actions/mongodb_explain_action_test.go

```diff
@@ -74,6 +79,82 @@ func TestMongoDBExplain(t *testing.T) {
 	})
 }
 
+// These tests are based on PMM-2.0 tests. The previous ones are inherited from PMM 1/Toolkit.
+func TestNewMongoDBExplain(t *testing.T) {
+	database := "sbtest"
+	id := "abcd1234"
+	ctx := context.TODO()
+
+	client := tests.OpenTestMongoDB(t)
+	defer client.Database(database).Drop(ctx) //nolint:errcheck
+
+	_, err := client.Database(database).Collection("people").InsertOne(ctx, bson.M{"last_name": "Brannigan", "first_name": "Zapp"})
+	require.NoError(t, err)
+
+	_, err = client.Database(database).Collection("orders").InsertOne(ctx, bson.M{"status": "A", "amount": 123.45})
+	require.NoError(t, err)
+
+	testFiles := []struct {
+		in         string
+		constraint string
+	}{
+		{
+			in: "distinct.json",
+		},
+		{
+			in:         "aggregate.json",
+			constraint: "3.6.0",
```

Comment:
Is that minimal version? Then I think we should call it `minVersion` or something. That should also simplify generic SemVer comparison function below.

Reply:
+1